### PR TITLE
Feature/blogpost categories

### PR DESCRIPTION
**Pull request by zalapeach**
## This pull request involves

* Make categories select works
* Fix select style for categories in mobile
*  Adds unit tests

## Description

Fix categories menu in mobile and add functionality

### CODE REVIEW

File: app/assets/javascripts/categories.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+pages.categories.mobileCategories = function(){
+  $('.categories-select').on('change', function(){
+    var category = $(this).find('option:selected').text().toLowerCase();
+    var url = '/categories/' +
+      $(this).val() +
+      '-' +
```
Comment (heridev) [inline app/assets/javascripts/categories.js]: @zalapeach why are you putting this string into a new separated line?,  I think there is enough space in the previous line to be there.

File: features/step_definitions/interaction_steps.rb [hunk 2]
```diff
@@ -1,7 +1,12 @@
 Given /^I click on "([^"]*)"$/ do |link_name|
+  #binding.pry
```
Comment (heridev) [inline features/step_definitions/interaction_steps.rb]: @zalapeach I think you dont need anymore this binding pry commented here :)

Comment (zalapeach) [inline features/step_definitions/interaction_steps.rb]: Yes, I already sent a commit for revert that

File: app/assets/javascripts/categories.js (hunk 1 above)
Comment (zalapeach) [inline app/assets/javascripts/categories.js]: For follow the ruby's style guide, let me change that

**Comment by heridev** [comment]
LGTM

**Pull request merged**